sources/niLang/src/DataTable.cpp

```diff
@@ -537,6 +538,15 @@ tU32 cDataTable::GetChildIndex(const achar *aaszName) const
   return eInvalidHandle;
 }
 
+///////////////////////////////////////////////
+tU32 cDataTable::GetChildIndexFromDataTable(iDataTable *apDT) const {
```

Comment:
`apDT->GetIndex()` does the same, so probably better to remove this.

Reply:
indeed